tests/stdlib/trandom.nim

```diff
@@ -1,21 +1,18 @@
 discard """
-  action: compile
+  joinable: false
 """
 
 import random
 
+randomize(233)
+
 proc main =
   var occur: array[1000, int]
 
   var x = 8234
   for i in 0..100_000:
     x = rand(high(occur))
     inc occur[x]
-  for i, oc in occur:
-    if oc < 69:
-      doAssert false, "too few occurrences of " & $i
-    elif oc > 150:
-      doAssert false, "too many occurrences of " & $i
```

Comment:
we should bring back this test, it's a good one, to prevent regressions.
this works:

```nim
import stats

...

  var rs: RunningStat
  for j in 1..5:
    for i in 1 .. 1_000:
      rs.push(gauss())
    doAssert abs(rs.mean-0) < 0.05
    doAssert abs(rs.standardDeviation()-1.0) < 0.1
    let maxVal = 3.0
    doAssert abs(rs.min + maxVal) < 1.0
    doAssert abs(rs.max-maxVal) < 1.0
    rs.clear()
```

Reply:
It is not working at my computer?

```
fatal.nim(49)            sysFatal
Error: unhandled exception: trandom.nim(22, 14) `abs(rs.mean - 0) < 0.05`  [AssertionDefect]
Error: execution of an external program failed: 'D:\QQPCmgr\Desktop\Nim\tests\stdlib\trandom.exe '
```